package.json

```diff
@@ -1,11 +1,8 @@
 {
-  "name": "ts-relayer",
-  "version": "0.0.3",
+  "name": "@confio/relayer",
+  "version": "0.1.0",
   "description": "IBC Relayer in TypeScript",
-  "main": "build/main/index.js",
-  "typings": "build/main/index.d.ts",
-  "module": "build/module/index.js",
```

Comment:
We don't have an entry point to the relayer (only binaries) so there is no reason to keep those.